game/platform.cc

```diff
@@ -23,6 +23,19 @@ Platform::Platform() {
 
 const std::array<const char*,6> Platform::platformNames = {{ "Windows", "Linux", "MacOS", "BSD", "Solaris", "Unix" }}; // Relevant for debug only.
 
+int Platform::defaultBackEnd() {
+		switch (Platform::currentOS()) {
+			case windows: return 13; // WASAPI
+			case macos: return 5; // CoreAudio
+			case solaris:
+			case bsd: return 7; // OSS
+			case linux:
```

Comment:
I see what you're trying to do here but cascading cases are always a bit confusing, possibly add some comments here to explain things more clearly.

Reply:
Or I can just explicitly add the return for each case; frankly I doubt there's any difference from an efficiency standpoint and it will look better.